python/test/test_api.py

```diff
@@ -359,8 +359,9 @@ def test_stop(self):
                         "parameters": {"decoupled": {"string_value": "False"}},
                         # Keep instance count low for fast startup/cleanup.
                         # Alternatively can use KIND_CPU here, but keeping gpus/count explicit.
+                        # TODO: Check above
                         "instance_group": [
-                            {"kind": "KIND_GPU", "gpus": [0], "count": 1}
+                            {"kind": "KIND_CPU"}
```

Comment:
This allows us to run this test on machines without GPUs, but the comment suggests it was intentionally run on GPU. If that's case, we will need to skip it in CI.

Reply:
This test isn't doing anything GPU specific 👍